src/plugin/sdl/sdl.c

```diff
@@ -998,6 +1025,176 @@ static void sdl_scrub(void)
   }
 }
 
+static void SDL_draw_string_utf8(void *opaque, int x, int y, unsigned char *text, int len, Bit8u attr)
```

Comment:
Confusing name, it doesn't take string in utf8.

Reply:
I followed the pattern in the X plugin, but I see your logic and renamed it to just SDL_draw_string